### PR TITLE
TIMOB-4523  Upgrade JavaScriptCore to 721.26

### PR DESCRIPTION
**Pull request by rseagraves**
This JavaScriptCore version is the one Apple shipped with iOS 4.3.3.

### CODE REVIEW

**Comment by sptramer** [comment]
Problems building under XCode 4.1/Lion. Please stand by... **HOLD**

**Comment by sptramer** [comment]
OK, turns out that this is because of a space in the pathname. There will be a separate fix for the issue.

Signed off on ability of the process to build. However, it looks like there are issues loading variables in the debugger. Going to indicate that on the TIMOB pull request, because it has nothing to do with the actual jscore - this pull request is **APPROVED**.

Still need signoff from Blain.

**Comment by sptramer** [comment]
Resolved crashes that were occurring in the debugger.  **APPROVED** on my end (again).

File: TiCore/TiCore.xcodeproj/.svn/all-wcprops [hunk 1]
```diff
@@ -0,0 +1,11 @@
```
Comment (WhichKatieDid) [inline TiCore/TiCore.xcodeproj/.svn/all-wcprops:0]: We probably shouldn't have SVN's leavings in the Repo.

**Comment by WhichKatieDid** [comment]
Functional test passed, with one caveat that should have further testing.
--EDIT: Further testing revealed this to be a bug in the sample code, and NOT in the code.

**Pull request merged**